x-pack/plugins/rule_registry/common/search_strategy/index.ts

```diff
@@ -12,8 +12,20 @@ import { IEsSearchRequest, IEsSearchResponse } from 'src/plugins/data/common';
 export type RuleRegistrySearchRequest = IEsSearchRequest & {
   featureIds: ValidFeatureId[];
   query?: { bool: estypes.QueryDslBoolQuery };
+  sort?: RuleRegistrySearchRequestSort[];
+  pagination?: RuleRegistrySearchRequestPagination;
 };
 
+export interface RuleRegistrySearchRequestPagination {
+  pageIndex: number;
+  pageSize: number;
+}
+
+export interface RuleRegistrySearchRequestSort {
```

Comment:
I think we should just use the `estypes.Sort` type from `import type * as estypes from '@elastic/elasticsearch/lib/api/typesWithBodyKey';`

Reply:
Done!